src/operator-docs/transformation/map.ts

```diff
@@ -1,6 +1,56 @@
 import { OperatorDoc } from '../operator.model';
 
 export const map: OperatorDoc = {
-  'name': 'map',
-  'operatorType': 'transformation'
+  name: 'map',
+  operatorType: 'transformation',
+  signature: 'public map(project: Function, thisArg: any): Observable',
```

Comment:
You could mark optional parameters with question mark:

public map(project: Function, thisArg?: any)

Reply:
There is an issue to discuss this https://github.com/ReactiveX/rxjs-docs/issues/168 so this PR should be good to go.